build.sbt

```diff
@@ -319,6 +320,22 @@ lazy val `kamon-cassandra` = (project in file("instrumentation/kamon-cassandra")
     )
   ).dependsOn(`kamon-core`, `kamon-instrumentation-common`, `kamon-testkit` % "test", `kamon-executors`)
 
+lazy val `kamon-elasticsearch` = (project in file("instrumentation/kamon-elasticsearch"))
+  .disablePlugins(AssemblyPlugin)
+  .enablePlugins(JavaAgent)
+  .settings(instrumentationSettings)
+  .settings(
+    fork in (Test,run) := true,
+    libraryDependencies ++= Seq(
+      kanelaAgent % "provided",
+      "org.elasticsearch.client" % "elasticsearch-rest-client" % "7.9.1" % "provided",
+      "org.elasticsearch.client" % "elasticsearch-rest-high-level-client" % "7.9.1" % "provided",
+      scalatest % "test",
+      logbackClassic % "test",
+      "com.dimafeng" % "testcontainers-scala_2.12" % "0.38.3" % "test",
+      "com.dimafeng" % "testcontainers-scala-elasticsearch_2.12" % "0.38.3" % "test"
```

Comment:
Why is the scala version included in these two artifacts? That will probably break cross testing.

Reply:
Yeah after some trying out, I'll include specific version for each of those libraries, depending on the version of scala used, e.g.
```
{
        if(scalaBinaryVersion.value == "2.11") {
          "com.dimafeng" % "testcontainers-scala-elasticsearch_2.11" % "0.38.3" % "test"
        } else if(scalaBinaryVersion.value == "2.12") {
          "com.dimafeng" % "testcontainers-scala-elasticsearch_2.12" % "0.38.3" % "test"
        } else {
          "com.dimafeng" % "testcontainers-scala-elasticsearch_2.13" % "0.38.3" % "test"
        }
      }
```